%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% "ModernCV" CV and Cover Letter
% LaTeX Template
% Version 1.1 (9/12/12)
%
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% Original author:
% Xavier Danaux (xdanaux@gmail.com)
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
% Important note:
% This template requires the moderncv.cls and .sty files to be in the same 
% directory as this .tex file. These files provide the resume style and themes 
% used for structuring the document.
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[11pt,letterpaper,sans]{moderncv} % Font sizes: 10, 11, or 12; paper sizes: a4paper, letterpaper, a5paper, legalpaper, executivepaper or landscape; font families: sans or roman

\moderncvstyle{classic} % CV theme - options include: 'casual' (default), 'classic', 'oldstyle' and 'banking'
\moderncvcolor{black} % CV color - options include: 'blue' (default), 'orange', 'green', 'red', 'purple', 'grey' and 'black'

\usepackage{etaremune}
\usepackage{doipubmed}
\usepackage[scale=0.85]{geometry} % Reduce document margins
%\setlength{\hintscolumnwidth}{3cm} % Uncomment to change the width of the dates column
%\setlength{\makecvtitlenamewidth}{10cm} % For the 'classic' style, uncomment to adjust the width of the space allocated to your name

%----------------------------------------------------------------------------------------
%	NAME AND CONTACT INFORMATION SECTION
%----------------------------------------------------------------------------------------

\firstname{Aroon} % Your first name
\familyname{Chande} % Your last name

% All information in this block is optional, comment out any lines you don't need
%\title{Curriculum Vitae}
\address{}{Atlanta, GA 30318}
\mobile{(515) 981-6868}
\email{mail@aroonchande.com}
\homepage{aroonchande.com}{https://aroonchande.com} 
\homepageII{aroonchan.de}{https://www.aroonchan.de} 

%----------------------------------------------------------------------------------------

\begin{document}

\makecvtitle % Print the CV title

%----------------------------------------------------------------------------------------
%	EDUCATION SECTION
%----------------------------------------------------------------------------------------

\section{Education}

\cventry{Aug 2017 -- present}{\textit{(In progress)} Ph.D. - Bioinformatics}{\textsc{Georgia Institute of Technology}}{Atlanta}{GA}{}  
\cventry{Dec 2016}{Master of Science - Bioinformatics}{\textsc{Georgia Institute of Technology}}{Atlanta}{GA}{}  
\cventry{May 2015}{Bachelor of Science - Microbiology}{\textsc{University of Iowa}}{Iowa City}{IA}{}
\cventry{May 2013}{Bachelor of Science - Biology}{\textsc{University of Iowa}}{Iowa City}{IA}{}


%----------------------------------------------------------------------------------------
%	WORK EXPERIENCE SECTION
%----------------------------------------------------------------------------------------

\section{Experience}

\cventry{May 2016 -- Present}{Bioinformatics Scientist}{\textsc{Applied Bioinformatics Laboratory}}{\small IHRC Inc}{Atlanta, GA}{
\textit{\httplink{http://abil.ihrc.com}}
\begin{itemize}
\item Development and maintenance of analytics platforms 
\item Development and maintenance of custom bioinformatics databases
\item Teaching bioinformatics courses
\end{itemize}
\textit{Languages used: C++/Bash/Julia/PHP/Perl/Python/R/SQL}}

\cventry{May 2016 -- Present}{Graduate Student}{\textsc{Jordan Lab}}{\small Georgia Institute of Technology}{Atlanta, GA}{
\textit{\httplink{http://jordan.biology.gatech.edu}}
\begin{itemize}
\item Studying the effects of human ancestry on disease burden in Colombia and South America 
\item Teaching and administrating classes in the Bioinformatics Masterâ€™s program
\item Development of bacterial typing schemes in collaboration with the Centers for Disease Control and Prevention
\end{itemize}
\textit{Languages used: Bash/Perl/Python/R/SQL/PHP/Julia/Rust}}

\cventry{Aug 2015 -- May 2016}{Graduate Student Research}{\textsc{Hammer Lab}}{\small Georgia Institute of Technology}{Atlanta, GA}{
\textit{\httplink{http://www.hammerlab.biology.gatech.edu/}}
\begin{itemize}
\item Development and maintenance of \textit{vibriocholera.com} 
\item \textit{Vibrio cholerae} NGS bioinformatic analysis
\item Development of webtool for Type VI Secretion System annotation
\end{itemize}
\textit{Languages used: Bash/Perl/R/Python}}

\cventry{Dec 2011 -- July 2015}{Research Assistant}{\textsc{Apicella Lab}}{\small University of Iowa}{Iowa City, IA}{
\begin{itemize}
\item Development of expression and purification protocols for NMR and x-ray crystallography study of \textit{Haemophilus influenzae} nuclease in \textit{E. coli} 
\item Screening and data collection on crystallized nuclease in-house and Synchrotron radiation (4.2.2 beamline at ALS, Berkeley)
\item Biological assays for protein activity, targets and pH dependence
\item Biofilm, immunoassay and confocal microscopy
\end{itemize}
\textit{Languages used: Bash/Perl}}

\cventry{May 2011 -- May 2013}{Protein Crystallography Core Staff}{Carver College of Medicine}{Unversity of Iowa}{Iowa City, IA}{
\begin{itemize}
\item Contracted protein purification and characterization
\item User training for protein expression, purification and crystallization 
\item Development of custom protocols for in-house robotic liquid handlers
\end{itemize}
\textit{Languages used: Bash}}

\pagebreak
\section{Skills}
\subsection{Computational}
\cvitem{Proficient}{Bash, Perl, Python, R, Unix systems administration}
\cvitem{Familiar}{HTML/CSS, SQL}
\cvitem{Learning}{Julia, Rust}
\cvitem{NGS}{Whole Genome Sequencing (WGS), Whole Exome Sequences (WES), RNA-seq, microarray}
\cvitem{'Omics}{Transcriptomics, metabolomics, proteomics}
\cvitem{Development}{- Bacterial typing and identification tools using NGS data \newline- Automated systems for bioinformatics analysis and information management}
\cvitem{Robotics}{Liquid handlers (Mosquito, TECAN, Alchemist platform),  Microscopy (Rock Imager), and other programmable robotics platforms}
\subsection{Wet Lab}
\cvitem{Pathogens}{Collection and handling of biological samples}
\cvitem{FPLC/HPLC}{Affinity column, ionic / hydrophobic chromatography}
\cvitem{Microscopy}{Confocal, Light, SEM}
\cvitem{Genetics}{PCR and cell line mutant development}


%----------------------------------------------------------------------------------------
%	INTERESTS SECTION
%----------------------------------------------------------------------------------------


\section{Posters and Presentations}

\begin{etaremune}
\item \textbf{Chande, A.T.}, Norris, E.T., Moreno, A.M., Nagar, S.D., Rishishwar, L., O'Neal, K., Velex, S., Montes-Rodrigues, S., Torres, I., Medina-Rivas, M.A., Valderrama-Aguirre, A., Jordan, I.K., Gallo, J.E. (2018) Population pharmacogenomics for precision public health in Colombia.  S{\~a}o Paulo Advanced School on Vaccines,  S{\~a}o Paulo, Brazil.
\item \textbf{Chande, A.T.}, Wang, L., Rishishwar, L., Conley, A.B., Norris, E.T., Valderrama-Aguirre, A., Jordan, I.K. (2018) GlobAl Distribution of GEnetic Traits (GADGET): Exploring polygenic trait scores.  The American Society of Human Genetics, San Diego, California.
\item \textbf{Chande A.T.}, Conley, A.B., Rowell, J., Rishishwar, L., Norris, E.T.,  Mari{\~{n}}o-Ram{{\'i}}rez, L., Valderrama-Aguirre, A.,  Medina-Rivas, M., Jordan, I.K. (2018) ChocoGen: Genetic ancestry and health in the Colombian Pacific. Closing the genomics research gap, McGill University, Montreal, Canada.
\item \textbf{Chande, A.T.}, Rishishwar, L., Watve, S., Jordan, I.K., Hammer, B.K. (2015). Genomic analysis of Type VI secretion systems of \textit{Vibrio cholerae}. 10th International Conference on Bioinformatics, Atlanta, GA.
\end{etaremune}

\section{Awards and Honors}
\cventry{Summer 2016}{Graduate Research Award}{\textsc{Jordan lab}}{Georgia Institute of Technology}{Alanta, GA}{Reclassification of Non-typeable \textit{Haemophilus spp.}.}

\section{Publications}

\begin{etaremune}
\item Bernardy, E.E., Petit, R.A., Moller, A.G., Blumenthal, J.A., McAdam, A.J., Priebe, G.P., \textbf{Chande, A.T.}, Rishishwar, L., Jordan, I.K., Read, T.D.,  Goldberg, J.B. (2019) Whole-Genome Sequences of Staphylococcus aureus Isolates from Cystic Fibrosis Lung Infections. Microbiol Resour Announc 8 (3), e01564-18 \doi{10.1128/MRA.01564-18} \pubmed{30687841 }
\item Crisan C.V.*, \textbf{Chande A.T.*}, Williams K., Raghuram V., Rishishwar L., Steinbach G., Yunker P., Jordan I.K., Hammer B.K. (2019) Analysis of \textit{Vibrio cholerae} genomes using a novel bioinformatic tool identifies new, active Type VI Secretion System gene clusters. bioRxiv. \doi{10.1101/526723}
\item \textbf{Chande, A.T.}, Wang, L., Rishishwar, L., Conley, A.B., Norris, E.T., Valderrama-Aguirre, A., Jordan, I.K. (2018) GlobAl Distribution of GEnetic Traits (GADGET) web server: polygenic trait scores worldwide. Nucleic Acids Research. \doi{10.1093/nar/gky415}. \pubmed{29788182}
\item Cho, C.*, \textbf{Chande, A.T.*}, Gakhar, L., Hunt, J., Ketterer, M. R., Apicella, M. A. (2018) Characterization of a nontypeable Haemophilus influenzae Thermonuclease. PLoS One. \doi{10.1371/journal.pone.0197010}. \pubmed{29746527}
\item Medina-Cordoba, L.K., \textbf{Chande, A.T}, Rishishwar, L., Mayer, L.W., Mari{\~{n}}o-Ram{{\'i}}rez, L., Valderrama-Aguirre, L., Valderrama-Aguirre, A., Kostka, J.E, Jordan, I.K. (2018). Genome sequences of 15 Klebsiella spp. isolates from sugarcane fields in Colombia's Cauca Valley. Genome Announcements. \doi{10.1128/genomeA.00104-18}. \pubmed{29567732}
\item \textbf{Chande, A.T.}, Rowell, J., Rishishwar, L., Conley, A.B., Norris, E.T., Valderrama-Aguirre, A., Medina-Rivas, M., Jordan, I.K. (2017). Influence of genetic ancestry and socioeconomic status on type 2 diabetes in the diverse Colombian populations of ChocÃ³ and Antioquia. Scientific Reports. \doi{10.1038/s41598-017-17380-4}. \pubmed{29215035}
\item Dale, R., Gr{\"u}ning, B., Sj{\"o}din, A., Rowe, J., Chapman, B. A., Tomkins-Tinch, C. H., . . . \textbf{Chande, A. T.}, . . . K{\"o}ster, J. (2017). Bioconda: A sustainable and comprehensive software distribution for the life sciences. bioRxiv. \doi{10.1101/207092}
\item Espitia, H, \textbf{Chande, A. T.}, Jordan, I. K., Rishishwar, L. (2017). Method of sequence typing with in silico aptamers from a Next Generation Sequencing platform. Patent Pending.
\item Post, D. M. B., Sl{\"u}tter, B., Schilling, B., \textbf{Chande, A. T}., Rasmussen, J. A., Jones, B. D., . . . Apicella, M. A. (2017). Characterization of Inner and Outer Membrane Proteins from Francisella tularensis Strains LVS and Schu S4 and Identification of Potential Subunit Vaccine Candidates. mBio, 8(5). \doi{10.1128/mBio.01592-17}. \pubmed{29018123}
\item Topaz, N., Mojib, N., \textbf{Chande, A. T.}, Kubanek, J., \& Jordan, I. K. (2017). RampDB: a web application and database for the exploration and prediction of receptor activity modifying protein interactions. Database, 2017, bax067-bax067. \doi{10.1093/database/bax067}. \pubmed{29220456}
\item Norris, E. T., Rishishwar, L., Wang, L., Conley, A. B., \textbf{Chande, A. T.}, Dabrowski, A. M., . . . Jordan, I. K. (2017). Assortative mating on ancestry-variant traits in admixed Latin American populations. bioRxiv. \doi{10.1101/177634}
\item Post, D. M. B., Schilling, B., Reinders, L. M., D'Souza, A. K., Ketterer, M. R., Kiel, S. J., . . . \textbf{Chande, A. T.}, . . . Gibson, B. W. (2017). Identification and characterization of AckA-dependent protein acetylation in Neisseria gonorrhoeae. PLoS One, 12(6), e0179621. \doi{10.1371/journal.pone.0179621}. \pubmed{28654654}
\item Watve, S. S., \textbf{Chande, A. T.}, Rishishwar, L., Mari{\~{n}}o-Ram{{\'i}}rez, L., Jordan, I. K., \& Hammer, B. K. (2016). Whole-Genome Sequences of 26 Vibrio cholerae Isolates. Genome Announcements, 4(6). \doi{10.1128/genomeA.01396-16}. \pubmed{28007852}
\item Cho, C.*,\textbf{ Chande, A.*}, Gakhar, L., Bakaletz, L. O., Jurcisek, J. A., Ketterer, M., . . . Apicella, M. A. (2015). Role of the Nuclease of Nontypeable Haemophilus influenzae in Dispersal of Organisms from Biofilms. Infection and Immunity, 83(3), 950-957. \doi{10.1128/iai.02601-14}.  \pubmed{25547799}
\item Apicella, M.A., Gakhar, L., \textbf{Chande, A.T.}  Novel Thermonuclease from Haemophilus Influenzae and Uses Thereof in Industry and Scientific Research. Patent Pending.
\end{etaremune}

\section{Manuscripts in preparation}

\begin{etaremune}
\item \textbf{Chande, A.T.}, Rishshwar, L., Conley, A.B.m Valderrama-Aguirre, A., Medina-Rivas, M., Jordan, I.K. (2019) Ancestry effects on type 2 diabetes genetic risk inference in Hispanic/Latino populations. Genes.
\end{etaremune}

\end{document}
